src/js/builtins/JSBufferPrototype.ts

```diff
@@ -317,13 +425,15 @@ export function writeUIntLE(this: BufferExt, value, offset, byteLength) {
       view.setUint16(offset + 4, Math.floor(value * 2 ** -32), true);
       break;
     }
-    default: {
-      throw new RangeError("byteLength must be >= 1 and <= 6");
-    }
   }
   return offset + byteLength;
 }
+
 export function writeUIntBE(this: BufferExt, value, offset, byteLength) {
+  require("internal/validators").validateIntRange(byteLength, "byteLength", 1, 6);
+  require("internal/validators").validateIntRange(offset, "offset", 0, this.byteLength - byteLength);
```

Comment:
`DataView` does it's own validation

Do we need to validate it twice?

Reply:
some but not all and none of the errors have codes